template/nestJs/endpoint.sh

```
@@ -1 +1 @@
-wait4x mysql "${DATABASE_USERNAME}:${DATABASE_PASSWORD}@tcp(mysql:3306)/${DATABASE_NAME}" --timeout=60s --interval=2s -- node migrate.js && node dist/main.js
+wait4x mysql "${DATABASE_USERNAME}:${DATABASE_PASSWORD}@tcp(mysql:3306)/${DATABASE_NAME}" --timeout=60s --interval=2s -- node ./migrate.js &&  node ./dist/main.js
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)